cg/server/api.py

```diff
@@ -467,6 +470,17 @@ def get_application_pipeline_limitations(tag: str):
     return jsonify([limitation.to_dict() for limitation in application_limitations])
 
 
+@BLUEPRINT.route("/orders")
+def get_orders():
+    """Return the latest orders."""
+    orders_request: OrdersRequest = OrdersRequest.model_validate(request.args)
```

Comment:
I'd move this logic to a separate function, maybe in `server/utils.py´ called `def parse_orders_request(request: Request) -> OrdersRequest`

Reply:
I think for me that does not add value since it would signal that it needs more manipulation than to be read into an object? Like I saw this in TB but I felt that this was clearer to me.

Reply:
Yeah, keep it 👍 Might make more sense later when we add pagination